cache/cache.go

```diff
@@ -52,6 +54,7 @@ type watchCtx struct {
 	backoffStart    time.Duration
 	backoffMax      time.Duration
 	onFirstResponse func() // callback to fire once on first upstream response
+	startRev        int64  // first revision to watch from (resp.Header.Revision+1)
```

Comment:
Let's not add impl. details as part of the comment, makes it hard to follow along. Either remove or just `first revision to watch from` is fine